api/core/database/models/api/bot/nomination_entry.py

```diff
@@ -19,7 +19,7 @@ class ApiNominationentry(Base):
     # The creation date of this nomination entry.
     inserted_at = Column(DateTime(True), nullable=False, default=datetime.now)
 
-    actor_id = Column(ForeignKey('api_user.id', deferrable=True, initially='DEFERRED'), nullable=False, index=True)
+    actor_id = Column(ForeignKey('user.id', deferrable=True, initially='DEFERRED'), nullable=False, index=True)
```

Comment:
Hmm, as far as I recall, the current site uses `api_user` for the table name, and we want to keep it compatible, right?

Reply:
I don't think so. Originally, I forgot to add `nomination_entry` to the models (since it was in the same files as the nominations and I did not see at first, but added later on in the PR), that's why I left the original naming unchanged `api_user`. However, in the API (and as far as I see in the site repo as well) the name is `User` or `user` respectively. 
That's the name the relationships were based on.